manifests/example-dev/core-modules.yaml

```diff
@@ -62,7 +62,13 @@ parameters:
 ---
 name: eks
 path: modules/core/eks/
+dataFiles:
+  - filePath: data/eks_dockerimage-replication/versions/1.25.yaml
+  - filePath: data/eks_dockerimage-replication/versions/default.yaml
 parameters:
+  - name: replicated-ecr-images-metadata
+    valueFrom:
+      parameterStore: /addf/demo-ssriche/replication/replication/metadata
```

Comment:
Change this to valueFrom -> ModuleMetadata from replications module and fetch the s3 key+file